pages/game_play_page.py

```diff
@@ -0,0 +1,96 @@
+from alttester import By
+
+from pages.base_page import BasePage
+
+
+class GamePlayPage(BasePage):
+
+    def __init__(self, altdriver, appium_driver):
+        super().__init__(altdriver, appium_driver)
+
+    @property
+    def pause_button(self):
+        return self.altdriver.wait_for_object(By.NAME, 'UICamera/Game/WholeUI/pauseButton', timeout=2)
+
+    @property
+    def character(self):
+        return self.altdriver.wait_for_object(By.NAME, 'PlayerPivot', timeout=2)
+
+    @property
+    def obstacles(self):
+        return self.altdriver.find_objects_which_contain(By.NAME, 'Obstacle')
+
+    def is_displayed(self):
+        return self.pause_button and self.character
+
+    def press_pause(self):
+        self.pause_button.tap()
+
+    def get_current_life(self):
+        return int(self.character.call_component_method('CharacterInputController', 'get_currentLife', assembly=''))
+
+    def jump(self):
+        self.character.call_component_method('CharacterInputController', 'Jump', assembly='')
+
+    def move_right(self):
+        self.character.call_component_method('CharacterInputController', 'ChangeLane', parameters=['1'])
```

Comment:
For all calls of `call_component_method` you should be updated from: 

```python 
call_component_method('CharacterInputController', 'get_currentLife', assembly='')
call_component_method('CharacterInputController', 'ChangeLane', parameters=['1'])
```

To: 

```python 
call_component_method('CharacterInputController', 'get_currentLife', '')
call_component_method('CharacterInputController', 'ChangeLane', '', parameters=['1'])
```